lua/lazyvim/plugins/extras/util/rest.lua

```diff
@@ -0,0 +1,28 @@
+vim.filetype.add({
+  extension = {
+    ["http"] = "http",
+  },
+})
+return {
+  {
+    "mistweaverco/kulala.nvim",
+    ft = "http",
+    keys = {
+      { "<leader>rs", "<cmd>lua require('kulala').run()<cr>", desc = "Send the request" },
+      { "<leader>rt", "<cmd>lua require('kulala').toggle_view()<cr>", desc = "Toggle headers/body" },
+      { "<leader>rp", "<cmd>lua require('kulala').jump_prev()<cr>", desc = "Jump to previous request" },
+      { "<leader>rn", "<cmd>lua require('kulala').jump_next()<cr>", desc = "Jump to next request" },
+    },
+    config = function()
+      require("kulala").setup({})
+    end,
+  },
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if type(opts.ensure_installed) == "table" then
+        vim.list_extend(opts.ensure_installed, { "http", "graphql" })
+      end
+    end,
```

Comment:
```suggestion
	opts = {
	  ensure_installed = { "http", "graphql" },
	},
```
`nvim-treesitter` is configured with [opts_extend = { "ensure_installed" }](https://github.com/LazyVim/LazyVim/blob/d35a3914bfc0c7c1000184585217d58a81f5da1a/lua/lazyvim/plugins/treesitter.lua#L25), so this isn't needed any more.